src/components/ListOfBooks.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
- [ ] For now please only have book.js in component directory. There should no other file.